meson.build

```diff
@@ -1,19 +1,46 @@
 project(
   'NumPy',
   'c', 'cpp', 'cython',
-  version: run_command(
-    # This should become `numpy/_version.py` in NumPy 2.0
-    ['numpy/_build_utils/gitversion.py'],
-    check: true).stdout().strip(),
-  license: 'BSD-3',
-  meson_version: '>=1.5.2',  # version in vendored-meson is 1.5.2
+  version: run_command(['numpy/_build_utils/gitversion.py'], check: true).stdout().strip(),
+  meson_version: '>=1.6.1',  # version in vendored-meson
   default_options: [
     'buildtype=debugoptimized',
     'b_ndebug=if-release',
     'c_std=c11',
     'cpp_std=c++17',
     'pkgconfig.relocatable=true',
   ],
+  # TODO: patch SPDX expression in PyPI wheel build process
+  license: 'BSD-3-Clause AND 0BSD AND MIT AND Zlib AND CC0-1.0',
+  license_files: [
+    'LICENSE.txt',                                       # BSD-3-Clause
+    'numpy/_build_utils/tempita/LICENSE.txt',            # BSD-3-Clause
```

Comment:
I don't think we bundle this with the wheel.

Reply:
Good catch, I missed that. Moved now, and same for the `spin` license file.